hs_communities/views/communities.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import absolute_import
+
+import json
+
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import Group, User
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.utils.html import mark_safe, escapejs
+from django.views.generic import TemplateView
+
+from hs_access_control.management.utilities import community_from_name_or_id
+from hs_access_control.models.community import Community
+from hs_communities.models import Topic
+
+
+class CollaborateView(TemplateView):
+    template_name = 'pages/collaborate.html'
+
+
+class CommunityView(TemplateView):
+    template_name = 'hs_communities/community.html'
+
+    def dispatch(self, *args, **kwargs):
+        return super(CommunityView, self).dispatch(*args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        # user_id = User.objects.get(pk=self.request.user.id)
+        grpfilter = self.request.GET.get('grp')
+
+        community_resources = community_from_name_or_id("CZO National Community").public_resources
+        groups = []
+        for c in community_resources:
+            if not any(str(c.group_id) == g.get('id') for g in groups):  # if the group id is not already present in the list
+                if c.group_name != "CZO National":  # The National Group is used to establish the entire Community
```

Comment:
Right now a community view has to be CZO?  Groups in a community are found by listing public resources in the CZO community and finding the set of groups from those resources... BUT it cannot include the "CZO National" group?

Reply:
Why can't you just use the member_groups property on the community model?